csrc/type.cpp

```diff
@@ -672,6 +667,8 @@ static const char* thread_size2string(ParallelType t) {
 
 const char* load_store_type2string(LoadStoreOpType t) {
   switch (t) {
+    case LoadStoreOpType::Automatic:
```

Comment:
"Automatic" seems like a confusing name since it's more like maybePermuted. Is it ok to print as a "Set" when there's a transpose?

Reply:
It is indeed confusing. I don't think there is anything we can do in `load_store_type2string`, but I changed `LoadStoreOp::toString` to print permute as a modifier. Now transpose is printed like below:
```C++
T3_l[ iS24{( ceilDiv(( i4 * i3 ), 8) )}, iS25{8} ] ca_pos( 1 )
   = Set.Permute( T2_g[ iS4{i3}, iS5{i4} ] )
```

Reply:
Why not change the enum name to `Set` as well?

Reply:
Yeah, let me change that.

Reply:
Changed. I somehow feel that `loadLocalToGlobal`, `loadGlobalToLocal`, ... should get their own `LoadStoreOpType`, but this is beyond the scope of this PR. Let's use `Set` in this PR and consider refactor later.